model/admin/db.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	Collection          = "admin"
-	systemSettingsDocID = "global"
+	SystemSettingsDocID = "global"
```

Comment:
I would try to keep these concepts separate, the value in model/admin is super important (though we could change it at some point,) and if we changed it, we probably wouldn't want to change the value in the event log?